content/en/docs/tasks/security/authorization/authz-td-migration/index.md

```diff
@@ -86,7 +86,7 @@ Notice that it may take tens of seconds for the authorization policy to be propa
 1. Install Istio with a new trust domain.
 
     {{< text bash >}}
-    $ istioctl install --set profile=demo --set values.global.trustDomain=new-td
+    $ istioctl install --skip-confirmation --set profile=demo --set values.global.trustDomain=new-td
```

Comment:
Can we use short notation wherever possible and keep it at the end?

```
$ istioctl install --set profile=demo --set values.global.trustDomain=new-td -y
```

Reply:
Will be picked up in a later cleanup commit, one tests pass. And since they are now passing, it's that time.

Reply:
Fixed